docker-entrypoint.sh

```diff
@@ -94,21 +94,33 @@ if [[ "$MONGODB_DATABASE" ]]; then
    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .mongodb.database=\"$MONGODB_DATABASE\""
 fi
 
-if [[ "$REDIS_HOST" ]]; then
+if [[ "$REDIS_SENTINEL_HOST" ]]; then
+    JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .localCache.name=\"sentinels\""
```

Comment:
So I think one problem is that the `name` needs to be set to zenko since this should be the redis cluster name. But really should be configurable from the env as well. `mymaster` is a typical redis default